requirements.txt

```diff
@@ -1,2 +1,12 @@
+beautifulsoup4==4.5.3
```

Comment:
I wonder if we want a separate test_requiements.txt – because it seems silly for a site deploy to install all these things that aren't needed.